JobConfig/cosmic/cosmic_defs_s1s2.fclinc

```diff
@@ -194,11 +193,11 @@ physics :
       cosmicFilter: 
       {
          module_type: FilterG4Out
-         noInstanceName : true
+         #noInstanceName : true
```

Comment:
The noInstanceName and numSimParticleCollections settings do not exist any more.  One can see the currently supported settings by running "mu2e --print-descriptions FilterG4Out".    Lisa, please remove such commented out lines for the obsolete parameters.

Reply:
I removed the outdated lines from these various fclinc files.